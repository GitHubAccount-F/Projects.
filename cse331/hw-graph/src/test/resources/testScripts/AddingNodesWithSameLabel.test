#This tests that when adding multiple nodes
#with the same label inside of a graph,
#it will only keep the first that was added
#and disregard any other attempts at adding a similar one

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

#Adds nodes to graphs
AddNode graph1 one
AddNode graph2 one
AddNode graph2 one
AddNode graph3 one
AddNode graph3 one
AddNode graph3 ONE

#Checks nodes inside of a graph
ListNodes graph1
ListNodes graph2
ListNodes graph3