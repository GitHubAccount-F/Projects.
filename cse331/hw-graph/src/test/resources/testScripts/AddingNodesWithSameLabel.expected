#This tests that when adding multiple nodes
#with the same label inside of a graph,
#it will only keep the first that was added
#and disregard any other attempts at adding a similar one

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3

#Adds nodes to graphs
added node one to graph1
added node one to graph2
added node one to graph2
added node one to graph3
added node one to graph3
added node ONE to graph3

#Checks nodes inside of a graph
graph1 contains: one
graph2 contains: one
graph3 contains: ONE one